docs/tutorials/kubernetes/efficient-data-sharing-in-kubeflow-with-vineyard-csi-driver.rst

```diff
@@ -277,6 +400,32 @@ As a result, we can find that with vineyard, the argo workflow
 duration of the pipeline is reduced by 10%~20% and the actual 
 execution time of the pipeline is reduced by about 30%.
 
+KFP V2 Result Analysis
+^^^^^^^^^^^^^^^^^^^^^^
+
+As the KFP V2 sets the ``image_pull_policy`` to ``Always`` by default, which means
+that the image will be pulled every time the pipeline is executed. However, after pulling
+the docker image to the kubelet node, the image will be cached. To avoid the cached image,
+we delete the run after each execution.
+
+The execution time of the pipeline shown in the kubeflow dashboard is as follows:
+
++------------+------------------+---------------+
+| data scale | without vineyard | with vineyard |
++============+==================+===============+
+| 8500 Mi    | 252s             | 228s          |
++------------+------------------+---------------+
+| 12000 Mi   | 299s             | 265s          |
++------------+------------------+---------------+
+| 15000 Mi   | 354s             | 316s          |
```

Comment:
Why kfp v2 ran much slower than kfp v1?

Reply:
The kfp v2 will set the `image_pull_policy` to `Always``by default, and the kfp v1 is `IfNotPresent`. Pulling docker images will slow down the process.

Reply:
Corrected here: Different with the KFP V1, the KFP V2 will compile the pipeline to the IR YAML,
and the IR YAML will be submitted to the kubeflow server. Before each component of the
pipeline is executed, the kubeflow server will create a preprocess container to parse
the IR YAML and generate the relevant kubernetes resources. That means the kubeflow server
will create more pods than the previous argo workflow, thereby slowing down the execution 
speed of the entire pipeline.